README.md

```diff
@@ -130,41 +141,40 @@ import into the policy library.
 To download some PDFs for development, you can run:
 
 ```bash
-docker-compose run --rm manage.py download_pdfs
+docker-compose run --rm api ./manage.py download_pdfs
 ```
 
 You can then access a variety of development and debugging-related
 views for the PDFs at http://localhost:8001/pdf/.
 
 ### Docker-compose commands
 
-There are two types of entry points:
-
-1. Services which will run until you press `ctrl-c`. These are activated via
-  `docker-compose up`
-  * `prod-api` - Build the admin/API app and run it in "production" mode on
-    port 9001
-  * `dev-api` - Build the admin/API app and run it in "development" mode on
-    port 8001
-  * `dev` - Build and run the UI and API app in "development" mode (port 8002
-    for UI, 8001 for API).
-  * `prod` - Run the UI and API apps in "production" mode (port 9002 for UI,
-    9001 for API). Note that this requires the JS be compiled already.
-1. One-use commands which run until complete. These are ran via
-  `docker-compose run --rm` (the `--rm` just deletes the images after running;
-  it's not strictly required)
-  * `api-npm`
-  * `api-webpack`
-  * `bandit`
-  * `flake8`
-  * `manage.py`
-  * `mypy`
-  * `npm`
-  * `pip-compile`
-  * `psql`
-  * `ptw`
-  * `py.test`
-  * `webpack`
+**Note:** You can add `--rm` to the following commands to delete the images
+after running the commands; alternatively, you can run
+`docker system prune` at regular intervals to do the same thing.
```

Comment:
IMO we should really remove `--rm` from all our examples and just tell users to either add it or regularly use `docker system prune`.  Having to type `docker-compose run <container-name>` before everything is already quite verbose, so the addition of the `--rm` makes our instructions seem unnecessarily more complex than they need to be, IMO.  But I figured this is out of scope of this PR so I didn't make the change.

Reply:
Where we can automate (e.g. wrapping scripts), we should, but I agree. My other dev environment is a netbook, so some of that is reflexive to me.